source/mod_dist.f90

```diff
@@ -98,159 +265,726 @@ subroutine dist_parseInputLine(inLine, iLine, iErr)
+  case("PZN")
+    call dist_appendFormat(dist_fmtPZN,      one,  6)
+
+  case("JX")
+    call dist_appendFormat(dist_fmtJX,       one,  1)
+  case("JY")
+    call dist_appendFormat(dist_fmtJY,       one,  3)
+  case("JZ")
+    call dist_appendFormat(dist_fmtJZ,       one,  5)
+  case("PHIX")
+    call dist_unitScale(fmtName, fmtUnit, 2, uFac, fErr)
+    call dist_appendFormat(dist_fmtPhiX,     uFac, 2)
+  case("PHIY")
+    call dist_unitScale(fmtName, fmtUnit, 2, uFac, fErr)
+    call dist_appendFormat(dist_fmtPhiY,     uFac, 4)
+  case("PHIZ")
+    call dist_unitScale(fmtName, fmtUnit, 2, uFac, fErr)
+    call dist_appendFormat(dist_fmtPhiZ,     uFac, 6)
+
+  case("MASS","M")
+    call dist_unitScale(fmtName, fmtUnit, 3, uFac, fErr)
+    call dist_appendFormat(dist_fmtMASS,     uFac, 0)
+  case("CHARGE","Q")
+    call dist_appendFormat(dist_fmtCHARGE,   one,  0)
+  case("ION_A")
+    call dist_appendFormat(dist_fmtIonA,     one,  0)
+  case("ION_Z")
+    call dist_appendFormat(dist_fmtIonZ,     one,  0)
+  case("PDGID")
+    call dist_appendFormat(dist_fmtPDGID,    one,  0)
+
+  case("4D") ! 4D default coordinates
+    call dist_appendFormat(dist_fmtX,        one,  1)
+    call dist_appendFormat(dist_fmtPX,       one,  2)
+    call dist_appendFormat(dist_fmtY,        one,  3)
+    call dist_appendFormat(dist_fmtPY,       one,  4)
+
+  case("6D") ! 6D default coordinates
+    call dist_appendFormat(dist_fmtX,        one,  1)
+    call dist_appendFormat(dist_fmtPX,       one,  2)
+    call dist_appendFormat(dist_fmtY,        one,  3)
+    call dist_appendFormat(dist_fmtPY,       one,  4)
+    call dist_appendFormat(dist_fmtZETA,     one,  5)
+    call dist_appendFormat(dist_fmtDPP0,     one,  6)
+
+  case("NORM") ! 6D normalised coordinates
+    call dist_appendFormat(dist_fmtXN,       one,  1)
+    call dist_appendFormat(dist_fmtPXN,      one,  2)
+    call dist_appendFormat(dist_fmtYN,       one,  3)
+    call dist_appendFormat(dist_fmtPYN,      one,  4)
+    call dist_appendFormat(dist_fmtZN,       one,  5)
+    call dist_appendFormat(dist_fmtPZN,      one,  6)
+
+  case("ACTION") ! 6D action
+    call dist_appendFormat(dist_fmtJX,       one,  1)
+    call dist_appendFormat(dist_fmtPhiX,     one,  2)
+    call dist_appendFormat(dist_fmtJY,       one,  3)
+    call dist_appendFormat(dist_fmtPhiY,     one,  4)
+    call dist_appendFormat(dist_fmtJZ,       one,  5)
+    call dist_appendFormat(dist_fmtPhiZ,     one,  6)
+
+  case("IONS") ! The ion columns
+    call dist_appendFormat(dist_fmtMASS,     c1e3, 0)
+    call dist_appendFormat(dist_fmtCHARGE,   one,  0)
+    call dist_appendFormat(dist_fmtIonA,     one,  0)
+    call dist_appendFormat(dist_fmtIonZ,     one,  0)
+    call dist_appendFormat(dist_fmtPDGID,    one,  0)
+
+  case("OLD_DIST") ! The old DIST block file format
+    call dist_appendFormat(dist_fmtPartID,   one,  0)
+    call dist_appendFormat(dist_fmtParentID, one,  0)
+    call dist_appendFormat(dist_fmtNONE,     one,  0)
```

Comment:
no, please, do not kill the parsing of the statistical weight.
In the context of the coupling, the statistical weight is currently operational

Reply:
Ah, I'll add it back in.

Reply:
What do you mean by "operational", in the old block the value was discarded. This is why I set it to NONE.